### PR TITLE
Fix default class

### PR DESCRIPTION
**Pull request by OlehSihunov**
## PR Submission Reminders:
- Include a jira ticket with number and description*
- Please keep the PR as small as possible, reviewer should be able to review in less than 15 minutes
- Check if your PR havs no breaking changes

### CODE REVIEW

**Review by andriy-mudrak — COMMENTED** [review]
row 10 Please change name from InitialScreen to LanguageScreen
row 12 this.props.deviceID==undefined. The code will not be executed. You can use console.log( this.props.deviceID) for debugging 
row 22  Doesn see ./images/images.png in Repository
row 26. Change from SetLang("en") to this.SetLang("en")
Please delete initialScreen  file. It is superfluous

File: Navigation/Navigator.js [hunk 1]
```diff
@@ -38,7 +37,7 @@ const DrawerNavigation = createStackNavigator(
 
 const InitialStack = createStackNavigator(
   {
-    initialScreen: { screen: LanguageScreen }
+    initialcreen: { screen: LanguageScreen }
```
Comment (andriy-mudrak) [inline Navigation/Navigator.js]: What is `initialcreen`? Use `initialScreen`

File: Screens/LanguageScreen.js [hunk 2]
```diff
@@ -19,17 +22,17 @@ export default class InitialScreen extends Component {
...
+          source={require('../img/images.png')}
         />
        </View>
        <View style={styles.buttonBox}>
-       <TouchableOpacity onPress={()=>SetLang("en")} style={styles.myButton}>
+       <TouchableOpacity onPress={()=>this.SetLang("en")} style={styles.myButton}>
           <Text style={{color:"white"}}>English</Text>
         </TouchableOpacity>
-        <TouchableOpacity onPress={()=>SetLang("ua")} style={styles.myButton}>
+        <TouchableOpacity onPress={()=>this.SetLang("ua")} style={styles.myButton}>
```
Comment (andriy-mudrak) [inline Screens/LanguageScreen.js]: We don`t have "ua" language. Change this to "uk".

**Review by taras-tymchuk — CHANGES_REQUESTED** [review]
Don't delete "baseURL.js.example". There are URL templates to the local and remote servers, so everyone can use it.
Make a copy of this file and rename it to "baseURL.js". There you can make your changes (because "baseURL.js" is in .gitignore).

**Review by vtorkalo — APPROVED** [review]

**Pull request merged**